pages/about/360tour.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Mount Sion CBS Secondary School</title>
+    <link href="../../semantic/out/semantic.min.css" rel="stylesheet" type="text/css">
+    <link href="../../assets/css/style.css" rel="stylesheet" type="text/css">
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js">
+    </script>
+    <script src="../../semantic/out/semantic.min.js">
+    </script>
+    <link href="../../assets/favicon.ico" rel="icon" type="assets/image/x-icon">
+      <script>
+          $(document).ready(function() {
+              $('#navbar').load('../../assets/html-navbar-footer/navigation.html');
+              $('#footer').load('../../assets/html-navbar-footer/footer.html');
+          });
+        </script>
+
+</head>
+<body>
+    <div class="ui divbg container">
+        <img class="ui centered image" height="128px" src="../../assets/images/logo.jpg">
+        <div id="navbar"></div>
+        <div class="ui padded basic segment">
+            <h1 class="ui block header center blue aligned">Virtual School Tour</h1>
+            <div class="ui center aligned blue segment">
+              <p>Welcome to our 360 Virtual School Tour!</p>
+              <p>This fantastic work was done by Sorin Sarba, a father of a pupil in our school. Sorin owns a business that you can check out <a href="_blank http://www.360primeview.ie">here</a>. Click around or use your arrow keys to find your way through our school.</p>
```

Comment:
@evanhearne the link isn't working for me. I think need to move the _blank to the "target" attribute rather than adding it into the "href"

Reply:
@laurafitzgerald thanks, will amend now.

Reply:
@laurafitzgerald please check again if changes work?